test-exchange-manager-values.yaml

```diff
@@ -0,0 +1,11 @@
+# Exchange Manager values to test.
+# This is a YAML-formatted file.
+
+exchangeManager:
```

Comment:
Can we enbale FTE in this test?

Reply:
Oops, my bad. 
Fixed the values file and enabled FTE.